rpm/sailfishos-chum-gui.spec

```diff
@@ -22,15 +22,25 @@ BuildRequires:  PackageKit-Qt5-devel
 BuildRequires:  qt5-qttools-linguist
 
 %description
-A client app for the Chum repositories
+A client app for the Chum repositories.
+
+PackageName: Chum GUI
+Type: desktop-application
+Categories:
+  - System
```

Comment:
This category is missing from new categories function ;)

Reply:
Ha, it is not. See https://github.com/sailfishos-chum/sailfishos-chum-gui/blob/main/qml/pages/CategoriesPage.qml#L104 - map is created by linking via IDs separated by `;`. As soon as package has one of the listed IDs, it will be included into that category